apps/pdc-frontend/src/components/OpenFormsEmbed/OpenFormsEmbed.scss

```diff
@@ -66,3 +67,34 @@
 .utrecht-form-container--openforms {
   flex: 1 1;
 }
+
+.openforms-formio-content {
+  &--info {
+    --of-formio-content-info-border-inline-start-style: none;
+    @include utrecht-alert;
+    @include utrecht-alert-type("info");
+
+    flex-direction: column;
+  }
+  &--warning {
+    --of-formio-content-warning-border-inline-start-style: none;
+    @include utrecht-alert;
```

Comment:
Ik denk dat je de verkeerde component extend, de spotlight section is voor content.

Reply:
Ik wilde eerst de spotlight-component gebruiken, maar die heeft twee varianten, namelijk 'info' en 'warning', terwijl in OpenFormulier vier varianten staan ('success', 'info', 'error', 'warning').

<img width="628" alt="Screenshot 2024-09-26 at 10 30 33" src="https://github.com/user-attachments/assets/682b5458-5c73-4799-8493-7d2890f0d070">

Reply:
Net overlegd met @Ollie-nl: Je mag de error mappen naar de warning spotlight. Dus dan zijn er twee manieren om de warning spotlight te krijgen.